src/wp-includes/compat.php

```diff
@@ -497,3 +497,35 @@ function str_ends_with( $haystack, $needle ) {
 if ( ! defined( 'IMG_WEBP' ) ) {
 	define( 'IMG_WEBP', IMAGETYPE_WEBP );
 }
+
+if ( ! function_exists( 'array_is_list' ) ) {
```

Comment:
Could we move this next to `array_key_first()` and `array_key_last()`, to have the array-related functions grouped together for a bit more consistent placement?